### PR TITLE
change irc:// URI to freenode's webchat

### PR DESCRIPTION
**Pull request by tejasmanohar**
most IRC clients don't seem to support irc://, obviously not the command-line ones. additionally, GitHub doesn't consider irc:// a hyperlink so that's less useful but it does hyperlink this (and we're on freenode)

### CODE REVIEW

**Comment by eksperimental** [comment]
I was the one who introduced that line of code.
(@jw2013 reported this a while ago but i cannot find the PR/Issue), 
the reason is that's a valid URI, regardless of the client you use supports that or or.
I was aware that Github doesn't support it now, but that doesn't mean it may not support it in the future. Also Github is not everything. I don't think we should removed things because github doesn't support it.

you can still read the README.md  file in your text editor, and understand that link and paste it if your client supports it, or maybe your command line viewer will link it automatically,

basically, It doesn't hurt to have.

**Comment by eksperimental** [comment]
oh, I didn't see you changed the link to the webchat...
that could work, but the name of the channel is wrong.

**Comment by tejasmanohar** [comment]
@eksperimental good point, and yeah I just fixed that... sorry! was testing how to get the ?channels working before- my bad.

**Comment by tejasmanohar** [comment]
@eksperimental I agree w/ you that the URI is valid though lots of clients don't support it and GitHub isn't everything, but I do think the webchat link is better since we're on Freenode and it's usable for people viewing this README on GitHub, which I imagine is where most of the viewership on that file is from.

It also indicates the #channel so provides mostly the same benefits. But yeah, no biggie- just noticed it.

**Comment by eksperimental** [comment]
@tejasmanohar sorry about that, I wrote that before i realized I had overlooked you changed irc URI to the webchat one.

**Comment by eksperimental** [comment]
@tejasmanohar: as i said in the my second msg, that could work. it adds a benefit. so i'm :+1:

**Comment by tejasmanohar** [comment]
:smile:

**Comment by lexmag** [comment]
@tejasmanohar please squash everything into one commit. :bowtie:

**Comment by tejasmanohar** [comment]
@lexmag First time squashing a commit... but based on the diff here, looks like it worked.

**Comment by tejasmanohar** [comment]
Thanks, @josevalim! Nice keynote last weekend.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:
